### ISSUE TITLE
Jekyll Build for Code Of Conduct appears to have failed

### ISSUE THREAD

**Issue by legodude17** [package-community/package-community.github.io#2]
The code of conduct currently looks like this: 
![screen shot 2017-06-10 at 5 41 31 pm](https://user-images.githubusercontent.com/11205096/27007316-1f62b3bc-4e04-11e7-8852-386f46f4218e.png)
That seems to indicate that the jekyll build failed and that GitHub is doing its default.

**Comment by zkat** [comment]
Yeah I'm not sure why it's doing this. It would be awesome if someone PRed a fix who knows more about Jekyll than I do
